google/cloud/forseti/common/gcp_api/groups_settings.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper for Groups Settings  API client."""
+from googleapiclient import errors
+from httplib2 import HttpLib2Error
+from google.auth.exceptions import RefreshError
+
+from google.cloud.forseti.common.gcp_api import _base_repository
+from google.cloud.forseti.common.gcp_api import api_helpers
+from google.cloud.forseti.common.gcp_api import errors as api_errors
+from google.cloud.forseti.common.gcp_api import repository_mixins
+from google.cloud.forseti.common.util import logger
+
+LOGGER = logger.get_logger(__name__)
+
+API_NAME = 'groupssettings'
+
+REQUIRED_SCOPES = frozenset([
+    'https://www.googleapis.com/auth/apps.groups.settings'
+])
+
+GSUITE_AUTH_FAILURE_MESSAGE = (
+    'Failed to retrieve G Suite Group Settings data due to authentication '
+    'failure. Please make sure your forseti_server_config.yaml '
+    'file contains the most updated information and enable G '
+    'Suite Groups Collection if you haven\'t done so. Instructions'
+    ' on how to enable: https://forsetisecurity.org/docs/latest/'
```

Comment:
Be consistent about where you put the space.  Should be at the end of the lines.